e2e/jest.test-setup.ts

```diff
@@ -202,7 +203,15 @@ const notRedirectRequest = (request: Request): boolean => {
 };
 
 const getResponseText = async (request: Request): Promise<string> => {
-  return (await request.response().buffer()).toString();
+  const response = request.response();
+  const status = response && response.status() ? response.status() : null;
+  // Explanation:
+  //  !(status > 299 && status < 400) -- it's not a redirect
+  //  !(status === 204) -- it's not a no-content response
```

Comment:
rather than comment, it may be clearer to add a constant in the file - something like:
```javascript
REDIRECT_CODE_START = 299;
REDIRECT_CODE_END = 400;
NO_CONTENT_RESPONSE_CODE = 204
```
Or a couple of functions:
```
isRedirect = ...
isNoContent = ...
```

Reply:
yeah. good call. I created `isNotContent` to handle 204 code. I restored `getResponseText` change. I forgot there is already `notRedirectRequest`.